autonomous/autonomous.py

```diff
@@ -122,13 +146,17 @@ def pick_up_cube(self, initial_call):
         # slowly ramp down the speed as we approach the cube
 
         speed = (self.PICKUP_SPEED - percent_along*(self.PICKUP_SPEED-0.5))
+        SmartDashboard.putNumber('cube_pickup_speed', speed)
+        SmartDashboard.putNumber('vision_angle', vision_angle)
+        SmartDashboard.putNumber('vision_alignment_heading', alignment_direction)
```

Comment:
AFAIK, calls to SmartDashboard.putNumber have a fairly large amount of overhead (compared to the alternatives which would give you the same result). Something to keep in mind.

Reply:
Which alternatives? The actual networktables API?

Reply:
The NetworkTables API, magicbot's NetworkTables integration… all of these give you NetworkTable entries.